libfwupd/fwupd-device.c

```diff
@@ -858,6 +858,73 @@ fwupd_device_set_modified (FwupdDevice *device, guint64 modified)
 	priv->modified = modified;
 }
 
+/**
+ * fwupd_device_subsume:
+ * @self: A #FwupdDevice
+ * @donor: Another #FwupdDevice
+ *
+ * Copy all properties from the donor object if they have not already been set.
+ *
+ * Since: 1.0.9
+ **/
+void
+fwupd_device_subsume (FwupdDevice *self, FwupdDevice *donor)
```

Comment:
If you don't like `subsume`, maybe `assimilate`?

Reply:
Maybe `incorporate`? Assimilate has negative associations.

Reply:
For more positive connotations, `embrace`?  Whichever you prefer though, no strong preference to me.